bridge/client/app/_views/ktb-environment-view/ktb-stage-overview/ktb-stage-overview.component.ts

```diff
@@ -17,65 +17,79 @@ import { ServiceFilterType } from '../ktb-stage-details/ktb-stage-details.compon
   templateUrl: './ktb-stage-overview.component.html',
   styleUrls: ['./ktb-stage-overview.component.scss'],
 })
-export class KtbStageOverviewComponent implements OnDestroy, OnInit {
-  public project?: Project;
-  public selectedStage?: Stage;
+export class KtbStageOverviewComponent implements AfterContentInit, OnDestroy {
   public _dataSource = new DtFilterFieldDefaultDataSource();
   public filter: DtFilterArray[] = [];
-  public isTriggerSequenceOpen = false;
+  public isTriggerSequenceOpen: boolean;
   private filteredServices: string[] = [];
   private globalFilter: { [projectName: string]: { services: string[] } } = {};
   private unsubscribe$: Subject<void> = new Subject<void>();
 
+  public project$ = this.route.params.pipe(
+    map((params) => params.projectName),
+    filter((projectName): projectName is string => !!projectName),
+    distinctUntilChanged(),
+    switchMap((projectName) => this.dataService.getProject(projectName)),
+    filter((project): project is Project => !!project),
+    tap((project) => {
```

Comment:
Be aware `tap()` can have side effects: If more than one is subscribing to this observable, the tap is executed multiple times